pkg/cri/annotations/annotations.go

```diff
@@ -45,6 +45,12 @@ const (
 	// workload can only run on dedicated runtime for untrusted workload.
 	UntrustedWorkload = "io.kubernetes.cri.untrusted-workload"
 
-	// containerName is the name of the container in the pod
+	// PodNamespace is the name of the namespace of the pod
+	PodNamespace = "io.kubernetes.cri.namespace"
+
+	// PodName is the name of the pod
+	PodName = "io.kubernetes.cri.name"
```

Comment:
could we change this to `pod-name`?  Just so it's clean by the label whenever you are viewing the container metadata and see the raw output?

Reply:
Let's use sandbox-name